analyses/proteomic-data/run_module.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash 
+
+set -e
+set -o pipefail
+
+## Run R script to substitue the sample_id in proteomic/phospho data to bs_id
+Rscript --vanilla 01-add-bsid-to-proteomic-data.R
+
+## Run python script to liftover the gene symbols in proteomic/phospho data 
+
+### whole cell proteomic data
+python 01-update_gene_symbols.py \
+  -g input/hgnc_complete_set.txt \
+  -f output/Hope_proteome_imputed_data.tsv \
+  -o output/Hope_proteome_imputed_data_liftover.tsv.gz \
+  -u "ApprovedGeneSymbol" \
+  --explode_records
+  
+### phospho data
+python 01-update_gene_symbols.py \
+  -g input/hgnc_complete_set.txt \
+  -f output/Hope_phosphosite_imputed_data_ischemia_removed.tsv \
+  -o output/Hope_phosphosite_imputed_data_ischemia_removed_liftover.tsv.gz \
+  -u "ApprovedGeneSymbol" \
+  --explode_records
+
+### phospho with motif data
+python 01-update_gene_symbols.py \
+  -g input/hgnc_complete_set.txt \
+  -f output/hope_phosphosite_imputed_data_ischemia_removed_motif.tsv \
```

Comment:
Might want to use a capital "H" for the output file names to match the output names above.